docs/zh/week03/03-3.md

```diff
@@ -4,5 +4,201 @@ lang-ref: ch.03-3
 title: Properties of natural signals
```

Comment:
The title is not translated!
Send a PR with this last bit, plz.
Just the title translation, not everything again.

Reply:
sry for forgetting the title. What's the meaning of not everything again?
and whether translator name attached? I found someone attached someone not. a little bit confused.

Reply:
Yes, add translator, translation-date, and title in Chinese.
Not everything, meaning you can send a PR with just a few lines edit. Do not submit the whole translation again. Just can also just click on "Edit" directly on GitHub, and change things there.